armory/scenarios/multimodal_so2sat_scenario.py

```diff
@@ -60,35 +60,42 @@ def _evaluate(
             estimator = load_defense_internal(config["defense"], estimator)
 
         attack_config = config["attack"]
-        attack_channels = attack_config.get("generate_kwargs", {}).get("channels")
+        attack_channels = attack_config.get("generate_kwargs", {}).get("mask")
```

Comment:
minor comment, but I think it'd be a little clearer if this variable name was something like `attack_channels_mask` rather than `attack_channels`. The former name I would expect implies a binary array (of length 14) while the latter name to me implies values ranging from (0,13) of length equal to the number of channels being attacked